akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/MdcLoggingDirectivesSpec.scala

```diff
@@ -0,0 +1,117 @@
+package akka.http.scaladsl.server.directives
+
+import akka.event.{ DiagnosticMarkerBusLoggingAdapter, MarkerLoggingAdapter }
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.RoutingSpec
+import scala.jdk.CollectionConverters._
+
+class MdcLoggingDirectivesSpec extends RoutingSpec {
```

Comment:
I'm wondering if there is a way to actually test logging output without making the tests project depend directly on logback-classic and akka-slf4j w/ a specific encoder? Maybe there's some mock testing logger that would let us inspect logs without actually going to the console?

Reply:
There's `EventFilter` in akka-testkit that can be used to intercept logging messages.

Reply:
👍 I added a test using EventFilter in my last commit.